servers/visual/visual_server_raster.h

```diff
@@ -68,27 +69,29 @@ class VisualServerRaster : public VisualServer {
 	List<FrameDrawnCallbacks> frame_drawn_callbacks;
 
 	void _draw_margins();
-	static void _changes_changed() {}
+	static void _changes_changed(int p_priority) {
+		if (p_priority) {
+			;
+		}
```

Comment:
This doesn't do anything, is that intended?

Reply:
Yes, fully intentional, it's to put the breakpoint on for finding what is triggering redraws. Maybe I can make this more clear with a comment. :+1:

Reply:
Would still be good to have a comment here to clarify that this isn't dead code.